cli/docs/flags.go

```diff
@@ -134,6 +135,9 @@ var commandFlags = map[string][]string{
 		url, user, password, accessToken, ServerId, SpecFlag, Threads, scanRecursive, scanRegexp, scanAnt,
 		Project, Watches, RepoPath, Licenses, OutputFormat, Fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly,
 	},
+	Enrich: {
+		url, user, password, accessToken, ServerId, SpecFlag, Threads,
```

Comment:
```suggestion
		url, user, password, accessToken, ServerId, Threads,
```
You are not supporting the `spec` flag, should be removed

Reply:
removed